Code/Editor/Core/QtEditorApplication.cpp

```diff
@@ -426,8 +429,16 @@ namespace Editor
 
     void EditorQtApplication::InstallEditorTranslators()
     {
-        m_editorTranslator =        CreateAndInitializeTranslator("editor_en-us.qm", ":/Translations");
-        m_assetBrowserTranslator =  CreateAndInitializeTranslator("assetbrowser_en-us.qm", ":/Translations");
+        QTextCodec::setCodecForLocale(QTextCodec::codecForName("utf-8"));
+
+        QStringList translatorNames = {
+            "Editor"
```

Comment:
What happened to the asset browser translator?

Reply:
If this is intended to be expanded in the future, i'd recommend exposing the list of translator names outside of the cpp file, maybe into the header file or inside the definition of this class as a constant list.

Reply:
We have added i18n to most modules and they will be added in later PR. Editor is just the first module we plan to tag and translate. Translators of the whole engine will include "Editor", "AssetProcessor",  "AzCore", "AzToolsFramework", "AzFramework", "AzQtComponents", "Legacy", ... , and every Gem.
I will move the list to a better position.

Reply:
Not all of the modules are linked against Qt. In general, all of the 'Editor' versions of the gems, AssetProcessor, and some tools will be linked against Qt, and will have access to the translators.